src/theme/provider.tsx

```diff
@@ -1,10 +1,17 @@
 import React from "react";
 import { ThemeProvider } from "@emotion/react";
 
-import theme from "./darkTheme";
+import darkTheme from "./darkTheme";
+import lightTheme from "./lightTheme";
 import GlobalStyle from "./globalstyle";
+import { Theme, useThemeQuery } from "@reearth/gql";
+import { useAuth } from "@reearth/auth";
 
 const Provider: React.FC = ({ children }) => {
+  const { isAuthenticated } = useAuth();
+  const { data } = useThemeQuery({ skip: !isAuthenticated });
+
+  const theme = data?.me?.theme === ("light" as Theme) ? lightTheme : darkTheme;
```

Comment:
```suggestion
  const theme = data?.me?.theme === Theme.LIGHT ? lightTheme : darkTheme;
```

In `src/gql/graphql-client-api.tsx`:

```ts
export enum Theme {
  Default = 'DEFAULT',
  Light = 'LIGHT',
  Dark = 'DARK'
}
```

Reply:
value is Returned and saved in back-end is in small-case like this "light"

Reply:
It's strange...